app/command.go

```diff
@@ -47,7 +47,7 @@ func (a *App) CreateCommandPost(post *model.Post, teamId string, response *model
 
 	if response.ResponseType == model.COMMAND_RESPONSE_TYPE_IN_CHANNEL {
 		return a.CreatePostMissingChannel(post, true)
-	} else if response.ResponseType == "" || response.ResponseType == model.COMMAND_RESPONSE_TYPE_EPHEMERAL {
+	} else if (response.ResponseType == "" || response.ResponseType == model.COMMAND_RESPONSE_TYPE_EPHEMERAL) && response.Text != "" {
```

Comment:
We should make sure this still works for responses that have attachments but no text.

Reply:
@ccbrown ah!! got it. will check

Reply:
@ccbrown  added extra check to make sure, after my conversation woth @jwilander